src/components/base/JInput/JInput.js

```diff
@@ -20,7 +20,7 @@ type Props = {
   isDisabled: boolean,
   isPinCode: boolean,
   isMultiline: boolean,
-  render?: Function
+  render: ?Function
```

Comment:
Do we need these props in defaultProps? Or they should be removed?

Reply:
When we have default prop with possible null value we should use "?Function" here
And, according this documentation https://flow.org/en/docs/react/components/#toc-using-default-props we don't need to use "render? " wen we have default prop